README.md

```diff
@@ -789,4 +790,27 @@ https://developer.apple.com/wwdc19/423
   - log out from iCloud
   - use release build of your app
   - measure warm launches
-  - measure launch with XCTest. Provides statistical results
\ No newline at end of file
+  - measure launch with XCTest. Provides statistical results
+  
+## Advances in Speech Recognition
+
+https://developer.apple.com/wwdc19/256
+
+* **Support for macOS** with 50+ languages which requires privacy approval from the user and Siri to be enabled.
+
+* **On-device speech recognition**, speech is private, stays on-device, network connection not required and no cellular data consumed. To utillise on-device learning set `requiresOnDeviceRecognition` on `SFSpeechRecognitionRequest` to `true`.
```

Comment:
Typo Should this be **utillise** ?

Reply:
Arn't both those spellings are the same?

Reply:
Oh okay I didn't know :)

Reply:
It does look funny, but that could also just be repeated looking at it :)